tests/test_pv_creation.py

```diff
@@ -15,7 +15,7 @@
 
 log = logging.getLogger(__name__)
 
-PV_YAML = os.path.join("templates/ocs-deployment", "PersistentVolume.yaml")
+PV_YAML = f"{TEMPLATE_PV_PVC_DIR}/PersistentVolume.yaml"
```

Comment:
```suggestion
PV_YAML = os.path.join(TEMPLATE_PV_PVC_DIR, "PersistentVolume.yaml")
```

Reply:
@vasukulkarni We should use `os.path.join()` here as well